### DISCUSSION TITLE
Fooocus using different drive(HDD) than its installation one (NVMe) when rendering

### DISCUSSION THREAD

**Discussion by mentalblasphemy** [lllyasviel/Fooocus#3610] [Q&A]
Hi.
I've got Fooocus installed on my NVMe because it made it work waay faster. But lately I've been reinstalling it and it slowed down. I've checked PC performance tab, and it seems it's using my old HDD for some reason. I thought it may be caching something in My Documents folder cuz it's the only one I still keep on my old good HDD storage disc, but I can't see anything. It goes to 100% usage and, beside slowing down the process, it slows down whole PC during work. Before it's been going superquick.

Edit:
I've noticed one more detail. It's using less of HDD if I start with animaPencil first. After that, whatever I do next it works way faster. But still - I have no idea why it's even touching HDD if nothing's there.

**Comment by Rogala** [comment]
Where is the swap file located?

**Reply by mentalblasphemy** [reply]
I answer in time :) It appeared I had it on the HDD but changing it to NVMe drive didn't help. Still running HDD. Same thing happen when I use Forge but it's happening for a way shorter moment so I don't feel it that much.
